index.js

```diff
@@ -10,6 +10,9 @@ server.use(express.json());
 
 let db = require('./data/db');  // const or let???
 
```

Comment:
Excellent way to ask questions Adrian! You have always been good at this. I have answered them below.